public/js/main.js

```diff
@@ -1,87 +1,106 @@
-var storageKey = 'allList';
+var storageKey = "all-list";
 
 var todoList = [];
-var allList = [];
-var doneList = [];
 
-var listString = '';
+var listString = "";
 
-function convertToHTML(list){
-  var content = allList.map(function(item){
-    return  `
-      <li>
-        <div class="view" id= ${item.id}>
-          <input class="toggle" type="checkbox" ${item.done?'checked':''}>
+function convertToHTML(list) {
+  var content = todoList.map(function (item) {
+    return `
+      <li id = ${item.id} class = ${item.done ? "completed" : ""}>
+        <div class = "todo-wrapper">
```

Comment:
Should not add spaces in a statement of html tag
Eg: `class="todo-wrapper"`

- [ ] remove spaces